### PR TITLE
New rotation fitness

### PR DESCRIPTION
**Pull request by portaloffreedom**

### CODE REVIEW

File: pyrevolve/data_analisys/visualize_robot.py [hunk 1]
```diff
@@ -47,16 +145,85 @@ async def test_robot_run(robot_file_path: str):
     robot.update_substrate()
     robot.save_file(f'{robot_file_path}.sdf', conf_type='sdf')
 
-    await connection.pause(False)
+    await connection.pause(True)
```
Comment (DaanZ) [inline pyrevolve/data_analisys/visualize_robot.py:148]: Change to False

File: pyrevolve/spec/msgs/body_pb2.py [hunk 2]
```diff
@@ -33,63 +33,64 @@
   filename=None,
   file=DESCRIPTOR,
   containing_type=None,
+  create_key=_descriptor._internal_create_key,
```
Comment (DaanZ) [inline pyrevolve/spec/msgs/body_pb2.py:36]: Are these just the generated files?

**Review by DaanZ — APPROVED** [review]

File: pyrevolve/spec/msgs/body_pb2.py (hunk 2 above)
Comment (portaloffreedom) [reply pyrevolve/spec/msgs/body_pb2.py:36]: yes, they are generated by the recompilation

File: pyrevolve/data_analisys/visualize_robot.py (hunk 1 above)
Comment (portaloffreedom) [reply pyrevolve/data_analisys/visualize_robot.py:148]: If you test a robot, do you want to immediately start moving or you want the prepare (start recording) and then click play to start the robot?

**Pull request merged**